### PR TITLE
improve docs about <a-animation begin="{event}"> (fixes #125)

### LINKED ISSUE #125: Improve Animation docs for `begin="{event}"`
https://aframe.io/docs/core/animation.html

### PR DESCRIPTION
**Pull request by cvan**
r? @ngokevin 

per question in A-Frame Slack #learning channel

### CODE REVIEW

File: src/docs/core/animation.md [hunk 1]
```diff
@@ -111,6 +113,14 @@ This can either be a number of milliseconds to delay or triggered by an event. I
 </a-entity>
 ```
 
+Or we could listen for when the element is hovered over:
```
Comment (ngokevin) [inline src/docs/core/animation.md]: `Or we define an animation that listens for when the element is hovered over` to match the previous sentence

**Comment by ngokevin** [comment]
The specific events listed only apply specifically with the cursor component. This seems like this should be part of the still unwritten "Adding interactivity" section.

**Comment by cvan** [comment]
> The specific events listed only apply specifically with the cursor component. This seems like this should be part of the still unwritten "Adding interactivity" section, or add notes about the cursor.

likely. but if it mentions "event name" but it doesn't give examples, I can see how this page would be confusing/incomplete to readers. had someone on Slack not know that it could be done, even after reading this page. do you want me to hold off on this until we have the "Adding Interactivity" section?

File: src/docs/core/animation.md (hunk 1 above)
Comment (cvan) [inline src/docs/core/animation.md]: better :+1:

**Comment by ngokevin** [comment]
It might be even more confusing listing event names that seem like they come out-of-the-box. If we add an example, we should add an agnostic one that show how and where those events are triggered. One that doesn't involve specifically the cursor component and one that doesn't imply it only works with click/mouse events. How about:

```html
<a-entity id="ball">
  <a-animation id="bounce" begin="start-bouncing" ...></a-animation>
</a-entity>
```

```js
document.querySelector('#ball').emit('start-bouncing');
```

**Comment by cvan** [comment]
> It might be even more confusing listing event names that seem like they come out-of-the-box.

well, `aframe-core` triggers those four events that I listed so they seem like good ones to mention. perhaps an explanation for each of when they're fired would suffice?

in which format would you like to see the possible event names listed? list? examples? sentence?

**Comment by ngokevin** [comment]
No, those events are only triggered when with the cursor component, which does not come by default. I am suggesting not listing any possible event names because there are none without any development, and using an example similar to the one above.

**Comment by cvan** [comment]
> No, those events are only triggered when with the cursor component which does not come by default. I am suggesting not listing any possible event names because there are none without any development, and using an example similar to the one above. After that, maybe we could add one example with the cursor component.

idk, people are pulling apart the examples and there are examples with a cursor. and people want to know how to do these things? not sure why we want to omit useful information. maybe it doesn't belong in this section. but this really ought to be information that we explicitly spell out. no point in withholding information we know about when we've already seen one individual ask us about this exact thing.

**Comment by ngokevin** [comment]
If you have the example I posted above and an example with the cursor, that's all the information.

I mentioned listing  `click`, `fusing`, `mouseenter`, `mouseleave` in the table would be *misleading* information.

**Comment by cvan** [comment]
sgtm

**Comment by ngokevin** [comment]
Docs are moved to: https://github.com/aframevr/aframe/blob/master/docs/core/animations.md#begin

Some mention of it at: https://github.com/aframevr/aframe/blob/master/docs/components/cursor.md#adding-visual-feedback

We could probably add more. Best handled by exposing the Pano Explorer example probably.

Tracking at https://github.com/aframevr/aframe/issues/1146

**Pull request closed without merging**